src/NServiceBus.Transport.SQS.AcceptanceTests/NServiceBus.Transport.SQS.AcceptanceTests.csproj

```
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net472;netcoreapp3.1;net5.0</TargetFrameworks>
+    <TargetFrameworks>net472;netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
```

We should remove `net5.0` or it will just make tests run long testing the extra framework. net6.0 is a LTS (long-term support) release, so net5.0 is kind of dead to us since it only has a few more months of support remaining.
```suggestion
    <TargetFrameworks>net472;netcoreapp3.1;net6.0</TargetFrameworks>
```